pod/enrichment/templates/enrichment/video_enrichment.html

```diff
@@ -10,34 +10,37 @@
 {% endif %}
 
-<meta property="og:site_name" content="{{ TITLE_SITE }}">
-<meta property="og:url" content="{% if request.is_secure %}https{% else %}http{% endif %}://{{ request.META.HTTP_HOST }}{% url 'enrichment:video_enrichment' slug=video.slug %}">
-<meta property="og:title" content="{{ video.title }}">
-<meta property="og:image" content="{% if request.is_secure %}https{% else %}http{% endif %}:{{ video.get_thumbnail_url }}">
-<meta property="og:image:secure_url" content="https:{{ video.get_thumbnail_url }}">
-<meta property="og:image:alt" content="{{ video.title }}">
-<meta property="og:image:width" content="{% if video.thumbnail.file_exist %}{{video.thumbnail.file.width}}{% else %}640{% endif %}" />
-<meta property="og:image:height" content="{% if video.thumbnail.file_exist %}{{video.thumbnail.file.height}}{% else %}360{% endif %}" />
-
-<meta property="og:description" content="{% if video.description or tag_list %}{{ video.description|metaformat|safe|striptags|truncatechars:250 }} {% if tag_list %}{% for tag in tag_list %}{{tag}} {%endfor%}%{% endif %}{% endif %} {% trans 'Added by' %}: {{ video.owner.get_full_name }}">
-
-<meta property="og:type" content="video">
-<meta property="og:video" content="{% if request.is_secure %}https{% else %}http{% endif %}://{{ request.META.HTTP_HOST }}{% url 'enrichment:video_enrichment' slug=video.slug %}?is_iframe=true">
-<meta property="og:video:secure_url" content="https://{{ request.META.HTTP_HOST }}{% url 'enrichment:video_enrichment' slug=video.slug %}?is_iframe=true">
-<meta property="og:video:type" content="video/mp4">
-<meta property="og:video:width" content="640">
-<meta property="og:video:height" content="{{video.get_player_height}}">
-
-<meta name="twitter:card" content="player">
-<meta name="twitter:site" content="{{ TITLE_SITE }}">
-<meta name="twitter:url" content="{% if request.is_secure %}https{% else %}http{% endif %}://{{ request.META.HTTP_HOST }}{% url 'enrichment:video_enrichment' slug=video.slug %}">
-<meta name="twitter:title" content="{{ video.title }}">
-<meta name="twitter:description" content="{% if video.description or tag_list %}{{ video.description|metaformat|safe|striptags|truncatechars:250 }} {% if tag_list %}{% for tag in tag_list %}{{tag}} {%endfor%}%{% endif %}{% endif %} {% trans 'Added by' %}: {{ video.owner.get_full_name }}">
-<meta name="twitter:image" content="{% if request.is_secure %}https{% else %}http{% endif %}:{{ video.get_thumbnail_url }}">
-
-<meta name="twitter:player" content="{% if request.is_secure %}https{% else %}http{% endif %}://{{ request.META.HTTP_HOST }}{% url 'video:video' slug=video.slug %}?is_iframe=true">
-<meta name="twitter:player:width" content="640">
-<meta name="twitter:player:height" content="{{video.get_player_height}}">
+{# Do not include metadata if video is restricted. #}
+{% if not form %}
```

Comment:
@Badatos la condition ne semble pas reflecter le titre de la PR.
je me serais attendu à voir quelque chose dans le genre
```html
{% if not video.is_restricted %}
<meta ...>
{% endif %}
```

Reply:
bonjour à tous, je plussoie l'avis de Eliam. Mais pour le coup, comme pour une card de video, je mettrai plutot {% if not video.password and not video.is_restricted %}

Reply:
En fait c’est plutôt le commentaire qui est erroné. Ce serait plutôt ´do not display meta until accès is géantes’

Reply:
hum oui tout à fait mais du coup, est-ce nécessaire d'ajouter les metadata si l'accès à la vidéo est réservé ?si oui alors il faut jsute modifier le commentaire, si non, alors il faut modifier la condition (et le commentaire ;) )